photon-api/src/test/scala/com/linkedin/photon/ml/function/glm/DistributedGLMLossFunctionTest.scala

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linkedin.photon.ml.function.glm
+
+import breeze.linalg.{DenseVector, Vector}
+import org.testng.Assert.assertEquals
+import org.testng.annotations.Test
+import org.apache.spark.sql.SparkSession
+
+import com.linkedin.photon.ml.data.LabeledPoint
+import com.linkedin.photon.ml.normalization.NoNormalization
+import com.linkedin.photon.ml.optimization._
+import com.linkedin.photon.ml.optimization.game.FixedEffectOptimizationConfiguration
+import com.linkedin.photon.ml.test.SparkTestUtils
+import com.linkedin.photon.ml.util.PhotonNonBroadcast
+
+/**
+ * Unit tests for [[DistributedGLMLossFunction]].
+ */
+class DistributedGLMLossFunctionTest {
+
+  import DistributedGLMLossFunctionTest._
+
+  /**
+   * Verify the value of loss function without regularization.
+   */
+  @Test()
+  def testValueNoRegularization(): Unit = {
```

Comment:
Please add a newline at the beginning of each test:
```
def testValueNoRegularization(): Unit = {

  val fixedEffect ...
```

Reply:
Done